### DISCUSSION TITLE
Command-Line output

### DISCUSSION THREAD

**Discussion by hvbtup** [ZUGFeRD/mustangproject#418] [Q&A] [answered]
I'm using Mustang-CLI-2.11.0 on the command-line to validate ZUGFeRD PDF or EN16931 XML files. 

Besides from a bug in VeraPDF regarding fonts, this works well, but I have doubts regarding the output.

It is full of irrelevant messages regarding the `cm` operator, and the `--no-notices` doesn't change it.

Part of the ouput is plain text, part is XML, as it seems.

Mustang writes messages to a file `ZUV-{date}.log` in the working directory.
There seems to be no option to configure this, and because I want to run several validation processes at the same time, there are going to be race conditions at best. 
One workaround for this could be to use a different working directory for each process, but are there better solutions?

Is it possible to write all the relevant messages only in a human-readable format into a file specified on the command line?

**Comment by jstaerk** [comment, marked as answer]
Hi,

that would be `--disable-file-logging`. I'm not sure if that also disables the console output but *that* output is for sure on stderr, so it can be redirected like `java -jar mustang.jar -Xmx1G --action validate --source xyz.pdf --disable-file-logging --no-notices 2>/dev/null`. There is a corresponding call for windows where I recommend to also set `-Dfile.encoding` to UTF-8.

kind regards
Jochen
